graphql/host_resolver.go

```diff
@@ -15,14 +15,17 @@ import (
 	"github.com/mongodb/grip/message"
 )
 
+// DistroID is the resolver for the distroId field.
```

Comment:
Did you run any special command to get these comments to generate or did you manually add them?

Reply:
I think they all appeared after running `make gqlgen`. I didn't add them manually.